opm/simulators/linalg/ISTLSolverEbos.hpp

```diff
@@ -273,8 +281,15 @@ namespace Opm
                 }
 #endif
 
+                if (numJacobiBlocks_ > 1) {
```

Comment:
Same here. cell_part will only be meaningful for serial run.

Reply:
0ea1f86